apm-collector/apm-collector-remote/collector-remote-grpc-provider/pom.xml

```diff
@@ -44,6 +44,11 @@
             <artifactId>apm-datacarrier</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>24.0-jre</version>
+        </dependency>
```

Comment:
Look like you add this just in last commit. https://github.com/apache/incubator-skywalking/pull/1506/commits/70b9bce97d4b4b5a150bd8d4c2fe1fc9cd6979b3

Reply:
Oh, I config agent-grpc and remote grpc in my application.yml, so it start need this.

Reply:
Guava-19 is already inside the release lib. So this may cause other issues. If you want this, try to add this:

```
            <dependency>
                <groupId>com.google.guava</groupId>
                <artifactId>guava</artifactId>
            </dependency>
```

The maven should do the rest.

Reply:
if not have this dependency, collector start failed, the message in log:
```
Exception in thread "main" java.lang.NoClassDefFoundError: com/google/common/base/Supplier
        at io.grpc.internal.AbstractServerImplBuilder.<clinit>(AbstractServerImplBuilder.java:58)
        at org.apache.skywalking.apm.collector.server.grpc.GRPCServer.initialize(GRPCServer.java:86)
        at org.apache.skywalking.apm.collector.grpc.manager.service.GRPCManagerServiceImpl.createOrChooseServer(GRPCManagerServiceImpl.java:64)
        at org.apache.skywalking.apm.collector.grpc.manager.service.GRPCManagerServiceImpl.createIfAbsent(GRPCManagerServiceImpl.java:43)
        at org.apache.skywalking.apm.collector.remote.grpc.RemoteModuleGRPCProvider.start(RemoteModuleGRPCProvider.java:76)
        at org.apache.skywalking.apm.collector.core.module.BootstrapFlow.start(BootstrapFlow.java:61)
        at org.apache.skywalking.apm.collector.core.module.ModuleManager.init(ModuleManager.java:68)
        at org.apache.skywalking.apm.collector.boot.CollectorBootStartUp.main(CollectorBootStartUp.java:45)
Caused by: java.lang.ClassNotFoundException: com.google.common.base.Supplier
        at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 8 more
```

Reply:
See `dependencyManagement` in `apm-collector/pom.xml`. The dependency has already existed.

Reply:
Ok. Thanks

Reply:
Oh,  It is my mistake that this pom was be commited. I have not excluded it when i operated commit.

Reply:
OK. Got it.

Reply:
> Oh, It is my mistake that this pom was be commited. I have not excluded it when i operated commit.

Need your another commit to revert this.

Reply:
Ok!